code/src/Main.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+
+import HappyForm from "./HappyForm"
+import HappyThougthList from "./HappyThougthList";
+
+
+
+const Main = () => { 
+const [thoughts, setThoughts] = useState([]); 
+
+const THOUGTHS_URL ="https://happy-thoughts-technigo.herokuapp.com/thoughts";
+
+    useEffect(() => {
+        fetchThougths();
+    }, []);  
+
```

Comment:
I've noticed that you're not very consistent with semi colons, maybe you can decide either to use it everywhere or nowhere ;p